_includes/footer.html

```diff
@@ -1,46 +1,18 @@
 <footer class="site-footer">
-
   <div class="wrapper">
-
     <h2 class="footer-heading">{{ site.title }}</h2>
 
     <div class="footer-col-wrapper">
       <div class="footer-col  footer-col-1">
         <ul class="contact-list">
           <li>Ping us on Twitter at <a href="https://twitter.com/austinonrails">@austinonrails</a></li>
-          <li><a href="mailto:{{ site.email }}">{{ site.email }}</a></li>
+          <li>Join us on <a href="https://aor-slack-invitebot.herokuapp.com/">Slack</a></li>
         </ul>
       </div>
 
       <div class="footer-col  footer-col-2">
-        <ul class="social-media-list">
```

Comment:
Getting rid of this section because we already link to our Twitter on the section right next to it, and I needed a space where to put the CF logo....